src/components/TextPopup/TextPopup.js

```diff
@@ -40,15 +40,15 @@ const TextPopup = () => {
   useEffect(() => {
     const textSelectTool = core.getTool('TextSelect');
     const onSelectionComplete = (startQuad, allQuads) => {
-      if (popupRef.current) {
+      if (popupRef.current && popupItems && popupItems.length > 0) {
```

Comment:
Do we actually need `&& popupItems &&`?

Reply:
Probably not because it looks like I'm getting `popupItems` from a selector and it looks like our Redux state initialize it to be [ ].

so it might be safe to just do `popupItems?.length > 0 ` ?

Reply:
If it’s always at least an empty array then I think we can access ‘length’ directly.

Reply:
done